lib/stats/DefaultStatsFactoryPlugin.js

```diff
@@ -458,25 +469,44 @@ const SIMPLE_EXTRACTORS = {
 			const chunks = compilationFileToChunks.get(asset.name) || [];
 			const auxiliaryChunks =
 				compilationAuxiliaryFileToChunks.get(asset.name) || [];
-			object.chunks = uniqueOrderedArray(chunks, c => c.ids, compareIds);
```

Comment:
@sokra Is it intended that this was removed? It seems that the `webpack-manifest-plugin` was relying on it.

Reply:
It has been moved to https://github.com/webpack/webpack/pull/9793/files/4c3d7c9cbb27329e3ced13767cca716661454701#diff-68fc93ae89cdcdc16d207fdcfd6e20b5R503